sdk/js/generated/api-definition.json

```diff
@@ -812,14 +812,14 @@
         "up.uplink_message.rx_metadata",
         "up.uplink_message.session_key_id",
         "up.uplink_message.settings",
-        "up.uplink_message.settings.coding_rate",
         "up.uplink_message.settings.concentrator_timestamp",
         "up.uplink_message.settings.data_rate",
         "up.uplink_message.settings.data_rate.modulation",
         "up.uplink_message.settings.data_rate.modulation.fsk",
         "up.uplink_message.settings.data_rate.modulation.fsk.bit_rate",
         "up.uplink_message.settings.data_rate.modulation.lora",
         "up.uplink_message.settings.data_rate.modulation.lora.bandwidth",
+        "up.uplink_message.settings.data_rate.modulation.lora.coding_rate",
         "up.uplink_message.settings.data_rate.modulation.lora.spreading_factor",
         "up.uplink_message.settings.data_rate.modulation.lrfhss",
```

Comment:
This file contains unrelated changes right?

Please see my review comment in #5715; this may break API

Reply:
This was indeed unrelated, when regenerating via `tools/bin/mage jsSDK:definitions` there seems to be some caching which caused this to remain. Running `tools/bin/mage jsSDK:definitionsClean jsSDK:definitions` was necessary to fix this.